src/main/java/edu/kit/ipd/crowdcontrol/objectservice/ExperimentController.java

```diff
@@ -0,0 +1,120 @@
+     * If an error occurs during the publishing, all already created tasks for this experiment will be undone
+     * @param experiment to be started
+     */
+   private void startExperiment(Experiment experiment){
+
+        for (int i = 0; i < experiment.getPopulationsCount(); i++) {
+            try {
+                platformManager.publishTask(experiment.getPopulations(i).getPlatformId(),experiment);
+
+            }catch ( TaskOperationException e1){
+                //could not create task
+                log.fatal(String.format("Error! Could not create experiment on platform %s!", experiment.getPopulations(i).getPlatformId()), e1);
+                unpublishExperiment(experiment);
+            }catch (IllegalStateException | IllegalArgumentException e2){
+                log.fatal("Error! Could not create experiment! "+e2.getMessage());
+                unpublishExperiment(experiment);
+            }
+        }
+
+        //ChangeEvent mit State.INVALID ?
+
+
+    }
+
+    /**
+     * Unpublishes an experiment from all platforms it is meant to be active on.
+     * This method is only called if the initialization of the experiment has failed on one
+     * of its platforms.
+     * @param experiment which is going to be unpublished
+     */
+    private void unpublishExperiment(Experiment experiment){
+        for (int i = 0; i < experiment.getPopulationsCount(); i++) {
+            try {
+                platformManager.unpublishTask(experiment.getPopulations(i).getPlatformId(),experiment);
+            }catch ( TaskOperationException e1){
+                //nothing to do here because experiment-publishing was aborted before this population was reached.
+            }
+        }
+    }
+
+
+    /**
+     * Unpublished the experiment from all its platform. Waits until the time for giving answers and/or ratings
+     * on the platforms (specified in the config-file) has run out. The experiment's state is set to STOPPED and a
+     * matching event is emitted.
+     * @param experiment which is to be ended.
+     */
+    public void endExperiment(Experiment experiment){
+
+        for (int i = 0; i < experiment.getPopulationsCount(); i++){
+            try {
+                platformManager.unpublishTask(experiment.getPopulations(i).getPlatformId(),experiment);
+            }catch (TaskOperationException e1){
+                log.fatal(String.format("Error! Cannot unpublish experiment from platform %s!",experiment.getPopulations(i).getPlatformId()),e1);
+            }
+        }
+        //wait for crowdplatform time out
+        try {
+            TimeUnit.HOURS.sleep(2);  //TODO get time from config
+        } catch (InterruptedException e) {
+            e.printStackTrace();
+        }
+        Experiment newExperiment = experiment.toBuilder().setState(Experiment.State.STOPPED).build();
+        //notify all observers
+        endExpObservable.emit(new ChangeEvent<>(experiment,newExperiment));
```

Comment:
this is going to emit a empty experiment with just a new state. @LeanderK we have to think about what to do here